Makefile

```diff
@@ -11,7 +11,7 @@ ifeq ($(DEV),true)
 endif
 
 serve:
-	docker run --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental $(CONFIG)
+	docker run --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental --no-watch $(CONFIG)
```

Comment:
Something we may want to pull out before merging, I think maybe you already plan on this but wanted to mention in case.